.github/workflows/pr-smoke-test.yml

```diff
@@ -0,0 +1,61 @@
+name: prcheck
```

Comment:
Sorry, @zhongruige one last thing. Let's give this a better name after the smoke test has completed:

<img width="300" alt="Screen Shot 2021-09-20 at 2 55 08 PM" src="https://user-images.githubusercontent.com/343847/134050722-a2d0bbcf-5aa4-457b-8aec-f119e3aa3664.png">

Reply:
Sure thing, I renamed it `Run smoke tests against pull request.`.